src/Microsoft.Azure.NotificationHubs/SRClient.Designer.cs

```diff
@@ -1211,18 +1247,7 @@ internal class SRClient {
                 return ResourceManager.GetString("FCMRegistrationInvalidId", resourceCulture);
             }
         }
-
```

Comment:
These values got nuked when I updated the resx file (they were not added there and this file is auto-generated). Will be fixed in a separate PR

Reply:
Fixed in #309